internal/pkg/manifest/workload.go

```diff
@@ -836,7 +836,7 @@ func DockerfileBuildRequired(svc interface{}) (bool, error) {
 	}
 	mf, ok := svc.(manifest)
 	if !ok {
-		return false, fmt.Errorf("manifest does not have required methods BuildRequired()")
+		return false, nil
```

Comment:
Is this ok to change, do we have any clients that rely on the error instead of checking the boolean?

Reply:
I tried to look at it and did not find any. And I think it's ok to combine "no need to build" and "do not apply" for now as the both results are "we can just skip".